golang/pkg/cli/runner.go

```diff
@@ -158,33 +162,21 @@ func StartContainers(containers *DyrectorioStack, settings *Settings) {
 	log.Info().Str("container", containers.Containers.MailSlurper.Name).Msg("started:")
 }
 
-// Cleanup for "down" command
-func StopContainers(containers *DyrectorioStack) {
-	ctx := context.Background()
+// Cleanup for "down" command, prefix can be provided with for multi removal
+func StopContainers(ctx context.Context, args *ArgsFlags) {
+	var prefixes string
 
-	containerNames := []string{
-		containers.Containers.MailSlurper.Name,
+	if args.Prefix != "" {
+		prefixes = args.Prefix
+	} else {
+		prefixes = util.JoinV(",", "dyo-stable", "dyo-latest")
 	}
 
-	if !containers.Containers.CruxUI.Disabled {
-		containerNames = append(containerNames, containers.Containers.CruxUI.Name)
-	}
-
-	if !containers.Containers.Crux.Disabled {
-		containerNames = append(containerNames,
-			containers.Containers.Crux.Name,
-			containers.Containers.CruxMigrate.Name)
-	}
-
-	containerNames = append(containerNames,
-		containers.Containers.KratosMigrate.Name,
-		containers.Containers.Kratos.Name,
-		containers.Containers.CruxPostgres.Name,
-		containers.Containers.KratosPostgres.Name,
-		containers.Containers.Traefik.Name)
+	time.Sleep(1 * time.Minute)
```

Comment:
😱What is this for?

Reply:
It was for testing purposes only, removing.